src/init.cpp

```diff
@@ -994,6 +995,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (GetBoolArg("-peerbloomfilters", true))
         nLocalServices |= NODE_BLOOM;
 
+    nMaxTipAge = GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
```

Comment:
It's  fine, as said it was a very minor nit. Maybe we can remove  the redundant initialization in main.cpp?
Whatever we do, I'm fine, as long as we try to be consistent  among all the globals (which imo should be in globals/server.o of something of the short instead of main chainparams, etc [globals/common.o could take the globals in unit.o and basechainparams, for example]; but one step a a time...).